tests/src/JIT/Directed/TypedReference/TypedReference.csproj

```diff
@@ -27,9 +27,10 @@
   <ItemGroup>
     <Compile Include="TypedReference.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <None Include="$(JitPackagesConfigFileDirectory)empty\project.json" />
-  </ItemGroup>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <RuntimeIdentifiers>win7-x86;win7-x64;ubuntu.14.04-x64;ubuntu.16.04-x64;ubuntu.16.10-x64;osx.10.12-x64;centos.7-x64;rhel.7-x64;debian.8-x64;fedora.24-x64</RuntimeIdentifiers>
```

Comment:
As part of the new project system these can all be parameterized now so instead of providing a huge list like this you can pass in exactly the one you want. That helps eliminate the need for keeping these updated as well as it cuts down on unnecessary restores of all RIDs on all platforms. This can be a clean-up task but I thought I would call it out.